pkg/virt-launcher/virtwrap/manager_test.go

```diff
@@ -427,9 +425,7 @@ var _ = Describe("Manager", func() {
 					Name:  "qemu",
 					Type:  "raw",
 				},
-				Alias: &api.Alias{
-					Name: "hpvolume1",
-				},
+				Alias: api.NewUserDefinedAlias("hpvolume1"),
```

Comment:
@rmohr While apply the `NewUserDefinedAlias`, I found these places were odd
- `ua-permvolume1` previously had the `ua-` spelled out. This had not happened in other places, only in this test.

Would this suggest that somewhere in kubevirt, the non-userdefined alias name maybe expected? 
- Converter.go defined 5 types of devices: disk, watchdog, input, interface, hostDev
- Now the most of the failures from rook-ceph and cnao are in networking_test.go and multus_test.go.